app/views/shared/_navbar.html.erb

```diff
@@ -0,0 +1,80 @@
+<nav id="navcontainer" class="navbar navbar-default navbar-fixed-top">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
```

Comment:
I don't *think* the `aria-expanded` properties are needed because this defaults to `false`.